webapp/components/wpt-results.html

```diff
@@ -326,6 +368,17 @@
         this.refreshDisplayedNodes();
       }
 
+      async fetchManifest(path) {
+        if (!this.computePathIsATestFile(path)) {
+          return;
```

Comment:
Shouldn't this throw so that the caller doesn't need to check the return value? (Looks like it's only used for an observer and I don't know how that works.)

Reply:
`fetchManifest` is an observer - it's just a callback for when something changes

Reply:
Oh, so this is async in order use await, not to return a promise. Got it.